src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -755,6 +757,11 @@ type PandaproxyAPITLS struct {
 	// duplicate the secret to the same namespace as redpanda CRD to be able to
 	// mount it to the nodes
 	NodeSecretRef *corev1.ObjectReference `json:"nodeSecretRef,omitempty"`
+	// If ClientCACertRef points to a secret containing the trusted CA certificates.
+	// If provided and RequireClientAuth is true, the operator uses the certificate
+	// in this secret instead of issuing client certificates. The secret is expected to provide
+	// the following keys: 'ca.crt'.
+	ClientCACertRef *corev1.TypedLocalObjectReference `json:"clientCACertRef,omitempty"`
```

Comment:
hmm, we should have named this `TrustedCACertsRef` instead. It is much more clear.